src/containers/ProfileDetails/components/RF/index.js

```diff
@@ -188,18 +188,20 @@ const RFForm = ({ form, details }) => {
           },
         })}
         {renderItem('Radio Mode', ['radioMode'], renderOptionItem, {
-          dropdown: (
-            <Select className={styles.Field}>
-              <Option value="modeN">N</Option>
-              <Option value="modeAC">AC</Option>
-              <Option value="modeGN">GN</Option>
-              <Option value="modeX">X</Option>
-              <Option value="modeA">A</Option>
-              <Option value="modeB">B</Option>
-              <Option value="modeG">G</Option>
-              <Option value="modeAB">AB</Option>
-            </Select>
-          ),
+          dropdown: key => {
+            return (
+              <Select className={styles.Field}>
+                <Option value="modeN">N</Option>
+                {key === 'is2dot4GHz' ? null : <Option value="modeAC">AC</Option>}
```

Comment:
Just put them all in one condition 
{ key === 'is2dot4GHz && <><Option ... /><Option ... /><Option ... /><Option ... /></> }